Sources/CwlViews/Implementations/iOS/CwlPickerView_iOS.swift

```diff
@@ -0,0 +1,370 @@
+			return nil
+		}
+
+		open func pickerView(_ pickerView: UIPickerView, viewForRow row: Int, forComponent component: Int, reusing view: UIView?) -> UIView {
+			return UIView()
+		}
+
+		open func pickerView(_ pickerView: UIPickerView, attributedTitleForRow row: Int, forComponent component: Int) -> NSAttributedString? {
+			return nil
+		}
+
+		open func pickerView(_ pickerView: UIPickerView, rowHeightForComponent component: Int) -> CGFloat {
+			return 0
+		}
+
+		open func pickerView(_ pickerView: UIPickerView, widthForComponent component: Int) -> CGFloat {
+			return 0
+		}
+
+		open func pickerView(_ pickerView: UIPickerView, didSelectRow row: Int, inComponent component: Int) {
+			return multiHandler(pickerView, PickerRowAndComponent(row, component))
+		}
+	}
+
+}
+
+// MARK: - Binder Part 6: BindingNames
+extension BindingName where Binding: PickerViewBinding {
+	public typealias PickerViewName<V> = BindingName<V, PickerView<Binding.ViewDataType>.Binding, Binding>
+	private static func name<V>(_ source: @escaping (V) -> PickerView<Binding.ViewDataType>.Binding) -> PickerViewName<V> {
+		return PickerViewName<V>(source: source, downcast: Binding.pickerViewBinding)
+	}
+}
+public extension BindingName where Binding: PickerViewBinding {
+	// You can easily convert the `Binding` cases to `BindingName` using the following Xcode-style regex:
+	// Replace: case ([^\(]+)\((.+)\)$
+	// With:    static var $1: PickerViewName<$2> { return .name(PickerView.Binding.$1) }
+	// 1. Value bindings may be applied at construction and may subsequently change.
+	static var showsSelectionIndicator: PickerViewName<Dynamic<Bool>> { return .name(PickerView.Binding.showsSelectionIndicator) }
+	static var pickerData: PickerViewName<Dynamic<PickerComponentMutation<PickerComponent<Binding.ViewDataType>>>> { return .name(PickerView.Binding.pickerData) }
+
+	// 2. Signal bindings are performed on the object after construction.
+	static var selectRowAndComponent: PickerViewName<Signal<SetOrAnimate<PickerRowAndComponent>>> { return .name(PickerView.Binding.selectRowAndComponent) }
+	static var reload: PickerViewName<Signal<PickerView<Binding.ViewDataType>.Reload>> { return .name(PickerView.Binding.reload) }
+
+	// 3. Action bindings are triggered by the object after construction.
+
+	// 4. Delegate bindings require synchronous evaluation within the object's context.
+	static var didSelectRowAndComponent: PickerViewName<(_ pickerView: UIPickerView, _ rowAndComponent: PickerRowAndComponent) -> Void> { return .name(PickerView.Binding.didSelectRowAndComponent) }
+	// Do we want to make the below have view re-use? (some how) *****************************
+	static var viewConstructor: PickerViewName<(_ rowSignal: SignalMulti<Binding.ViewDataType>) -> ViewConvertible> { return .name(PickerView.Binding.viewConstructor) }
+
+	// Composite binding names
+	static var rowSelected: PickerViewName<SignalInput<PickerRowAndComponent>> {
```

Comment:
I am not necessarily sold on only returning the most recently picked element. Which is the default UIPickerView behaviour. It would be nice to return each selected row from the all the components.

Reply:
I am also only returning the index, vs returning the actual element. Which is the nice behaviour we have on the table view. This means if users of this wrapper want the actual element they have to do that binding on their 'end'.